server/src/test/java/org/opentosca/toscana/plugins/testdata/LampApp.java

```diff
@@ -0,0 +1,159 @@
+package org.opentosca.toscana.plugins.testdata;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.opentosca.toscana.model.EffectiveModel;
+import org.opentosca.toscana.model.capability.AdminEndpointCapability;
+import org.opentosca.toscana.model.capability.AttachmentCapability;
+import org.opentosca.toscana.model.capability.BindableCapability;
+import org.opentosca.toscana.model.capability.ContainerCapability;
+import org.opentosca.toscana.model.capability.DatabaseEndpointCapability;
+import org.opentosca.toscana.model.capability.EndpointCapability;
+import org.opentosca.toscana.model.capability.OsCapability;
+import org.opentosca.toscana.model.capability.Requirement;
+import org.opentosca.toscana.model.capability.ScalableCapability;
+import org.opentosca.toscana.model.datatype.Port;
+import org.opentosca.toscana.model.datatype.Range;
+import org.opentosca.toscana.model.node.Apache;
+import org.opentosca.toscana.model.node.BlockStorage;
+import org.opentosca.toscana.model.node.Compute;
+import org.opentosca.toscana.model.node.MysqlDatabase;
+import org.opentosca.toscana.model.node.MysqlDbms;
+import org.opentosca.toscana.model.node.RootNode;
+import org.opentosca.toscana.model.node.WebApplication;
+import org.opentosca.toscana.model.operation.Operation;
+import org.opentosca.toscana.model.operation.OperationVariable;
+import org.opentosca.toscana.model.operation.StandardLifecycle;
+import org.opentosca.toscana.model.relation.AttachesTo;
+import org.opentosca.toscana.model.relation.HostedOn;
+
+public class LampApp {
```

Comment:
This class needs to get refactored. Please don't use these comment dividers between logical chunks. There is a quite nice language feature in java called `methods`. Check it out :1st_place_medal: 
These so called `methods` will later be extremely useful: Correctly used, they offer great reusability value